static/docs/commands-reference/status.md

```diff
@@ -6,12 +6,15 @@ cache and local files, or between the local cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-q | -v] [-j JOBS] [--show-checksums]
-                  [-c] [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c] [-r REMOTE]
+                  [-a] [-T] [-d]
                   [targets [targets ...]]
 
+Show changed states, compare local cache and remote storage.
```

Comment:
no need in this change, we don't include it usually. A summary at the very top of the page is enough. Check other command - like checkout fr example.

Reply:
Okay, I will remove these changes.